01-numpy.html

```diff
@@ -68,17 +68,23 @@ <h2 id="learning-objectives" class="objectives objectives">Learning Objectives</
 <pre class="output"><code>weight in kilograms is now: 57.5</code></pre>
 <p>As the example above shows, we can print several things at once by separating them with commas.</p>
 <p>If we imagine the variable as a sticky note with a name written on it, assignment is like putting the sticky note on a particular value:</p>
-<p><img src="img/python-sticky-note-variables-01.svg" alt="Variables as Sticky Notes" /></p>
+<div class="figure">
+<img src="fig/python-sticky-note-variables-01.svg" alt="Variables as Sticky Notes" /><p class="caption">Variables as Sticky Notes</p>
```

Comment:
@r-gaia-cs, this is what I mean by the caption. It adds a `<p class="caption">` to the HTML in addition to the `alt` attribute.